### PR TITLE
Adds v4 constraint to the browser

### PR DESCRIPTION
**Pull request by rileyhgrant**
Updates help page text for v4 constraint

Updates app metadata to render constraint tables on v4

Accompanied by a data update of the gene_grch38 and transcripts_grch38 indices after a re-run of the genes pipeline to re-annotate those indices with the new constraint files

### CODE REVIEW

File: browser/help/topics/constraint.md [hunk 1]
```diff
@@ -3,50 +3,69 @@ id: constraint
...
+  - [LOEUF: loss-of-function observed / expected upper bound fraction](/help/constraint#loeuf)
+  - [pLI: probability of being loss-of-function intolerant](/help/constraint#pli)
+  - [Z score](/help/constraint#z-score)
+- [An explanation of “What is the difference between the LOEUF and pLI score?](/help/constraint#loeuf-vs-pli)”
 
-The constraint score shown in gnomAD is the ratio of the observed / expected (`oe`) number of loss-of-function variants in that gene. The expected counts are based on a mutational model that takes sequence context, coverage and methylation into account.
+More details on these methods can be found in the supplement of [Lek _et al._ Nature 2016](https://www.nature.com/articles/nature19057) and [Karczewski _et al._ Nature 2020](https://www.nature.com/articles/s41586-020-2308-7).
 
-#### Interpretation
+### <a id="methods"></a> Methods
```
Comment (sjahl) [inline browser/help/topics/constraint.md:19]: are these just linkified for anchoring?

**Review by sjahl — APPROVED** [review]
Just one question inline on the links, since they looked odd in the markdown. Otherwise this looks good to me. I'm assuming the content of the FAQ/other text updates has been vetted. I mostly checked for broken links, spelling, formatting issues, and didn't come across any.

Perused through the snapshots, and things seem to go as expected there

File: dataset-metadata/metadata.ts
Comment (elissa-alarmani) [inline dataset-metadata/metadata.ts:1]: Do we need to include `startsWith(gnomad_cnv_4)` to the `hasConstraints`? Or is that covered somewhere else?

**Review by elissa-alarmani — APPROVED** [review]
Looks good to me, just left one question about the cnv `datasetId`

File: browser/help/topics/constraint.md (hunk 1 above)
Comment (rileyhgrant) [reply browser/help/topics/constraint.md:19]: Yeah exactly, just for anchoring.

File: dataset-metadata/metadata.ts
Comment (rileyhgrant) [reply dataset-metadata/metadata.ts:1]: Good q, I think we actually can do the opposite and remove `startsWith(gnomad_sv_r4)`, as the `includes(r4)` should cover v4, svs v4, and cnvs v4, however I don't want to change very many things, and pretty much just exactly reverted these lines as to how they were before I removed the v4 checks on `hasConstraints`

So thats just to say, good eye, and I'd prefer to change as few things as possible to get this v4 constraint out, and revisit this metadata file shortly (probably with some type of overhaul that converts them to object level metadata)

**Pull request merged**